Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -83,6 +88,25 @@ static void Read()
   }
 }
 
+static void Write()
+{
+  int size = 0;
+
+  while (true)
+  {
+    s_rumble_data_available.Wait();
+
+    if (s_adapter_thread_running.IsSet())
```

Comment:
Very minor, but returning early is preferable to avoid extra levels of indentation